bundle.yaml

```diff
@@ -36,6 +36,21 @@ applications:
     channel: "stable"
     scale: 1
 
+  nginx-ingress-integrator-sdlc:
```

Comment:
```suggestion
  sdlc-ingress:
```

We could consider some shorter names here?

Reply:
On this, we could rename a bunch of these charms so you get:

- mongodb
- legend-studio
- legend-sdlc
- legend-engine
- gitlab-integrator
- legend-db

Just makes typing juju commands a bit shorter! Thoughts @pedroleaoc / @balbirthomas ?

Reply:
No, changing the name at the top level in the bundle is just like deploying a charm with an alias: `juju deploy mongodb-k8s mongodb`, for example

Reply:
Done